### DISCUSSION TITLE
Regex help

### DISCUSSION THREAD

**Discussion by shunt3090** [SystemJargon/pi-hole#2] [Q&A] [answered]
Is there any documentation that we can strip out port numbers from lists or similar?

**Comment by SystemJargon** [comment, marked as answer]
> Is there any documentation that we can strip out port numbers from lists or similar?

Yes refer to [this guide](https://github.com/SystemJargon/regex/blob/main/regex-urls-domains.md) if not covered by [this](https://github.com/SystemJargon/pi-hole/blob/main/regex/README.md)
